nx-X11/include/extensions/Imakefile

```diff
@@ -42,15 +42,6 @@ XINERAMAHEADERS = Xinerama.h panoramiXext.h panoramiXproto.h
 #if BuildRender || BuildXftLibrary
 RENDERHEADERS = render.h renderproto.h
 #endif
-#if BuildXfixes || BuildXfixesLibrary
-XFIXESHEADERS = xfixeswire.h xfixesproto.h
-#endif
-#if BuildDamage || BuildDamageLibrary
-DAMAGEHEADERS = damagewire.h damageproto.h
-#endif
-#if BuildComposite || BuildCompositeLibrary
-COMPOSITEHEADERS = composite.h compositeproto.h
-#endif
```

Comment:
Due to your review I more clearly see the differences between lib and proto part of the Xserver.

I will re-add all proto-related stuff to this PR and update it (to latest X.org or so!) later with another PR.